### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by sashax**
Add all the templates except FIL. HTML checks out in general, but I'm not sure how to test them in email context.

### CODE REVIEW

**Comment by hshaosf** [comment]
Hm. I don't think we are sending phase information over. I created a separate [issue](https://github.com/SFDigitalServices/vaccination-distribution/issues/36) to track that.

File: service/templates/email/confirmation_es.txt [hunk 1]
```diff
@@ -1 +1,6 @@
-Hola {{ data.name }}, gracias!
\ No newline at end of file
+{{ data. name }}
```
Comment (hshaosf) [inline service/templates/email/confirmation_es.txt:1]:
```suggestion
{{ data.name }}
```

File: service/templates/email/confirmation_future_es.txt [hunk 2]
```diff
@@ -1 +1,8 @@
-Hola {{ data.name }}, gracias!
\ No newline at end of file
+{{ data. name }}
```
Comment (hshaosf) [inline service/templates/email/confirmation_future_es.txt:1]:
```suggestion
{{ data.name }}
```

**Review by hshaosf — CHANGES_REQUESTED** [review]
some minor typos.

File: service/templates/email/confirmation_es.txt (hunk 1 above)
Comment (sashax) [reply service/templates/email/confirmation_es.txt]: thanks! Fixed

File: service/templates/email/confirmation_future_es.txt (hunk 2 above)
Comment (sashax) [reply service/templates/email/confirmation_future_es.txt]: thanks! Fixed.

**Review by hshaosf — APPROVED** [review]

**Pull request merged**